plugin.video.gq/resources/language/English/strings.xml

```diff
@@ -1,37 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any reason why you choose to remove this and hardcode instead of converting it to PO?

Reply:
Not really. There used to be a bunch of strings used and now there are only a couple: next, new, popular. The source website is English only so it really doesn't make sense to fool around with translations.
Probably the most pertinent excuse: I'm pretty lazy.